src/__tests__/deployments.helper.ts

```diff
@@ -86,7 +86,12 @@ export function getVersionsByChainIdByDeploymentMap(): VersionsByChainIdByDeploy
 
       // Get the alias name
       const name = Object.entries(deploymentAliases).find(([, aliases]) => {
-        return aliases.includes(deployment.contractName);
+        if (typeof deployment.contractName === 'string') {
+          const contractName: string = deployment.contractName;
+          return aliases.includes(contractName);
+        } else {
+          throw new Error('contractName must be a string');
+        }
```

Comment:
Nit: we can reduce the code with an early return removing the unnecessary assignment, e.g.

```suggestion
        if (typeof deployment.contractName !== 'string') {
          throw new Error('contractName must be a string');
        }
        return aliases.includes(deployment.contractName);
```